website/community/product-steering/_index.md

```diff
@@ -35,7 +35,7 @@ The committee is elected by the project's active contributing sponsors and consi
 - Others are encouraged to **watch the recording** instead of attending live.
 
 #### 3. Meeting Process
-- The proposer presents the product proposal.
+- The proposer presents the product proposal including a clear decision ask (if necessary).
```

Comment:
Can you maybe list some examples here?

Reply:
Does the example help.
WDYT?

Reply:
Sounds good for now
cc @marwinski any other proposals?

Reply:
> Sounds good for now cc @marwinski any other proposals?

Should I merge the PR or wait for further feedback?